packages/language-server/src/core/astro2tsx.ts

```diff
@@ -109,7 +109,7 @@ function getVirtualFileTSX(
 										semantic: true,
 										navigation: true,
 										structure: true,
-										format: true,
+										format: false,
```

Comment:
Unrelated fix, but we don't want TSX formatting since we handle it all in Prettier